### DISCUSSION TITLE
Changing export full backup associated app?

### DISCUSSION THREAD

**Discussion by steel-steven-87** [iSoron/uhabits#1745] [Help & FAQ]
I use Syncthing to export full backup periodically but recently something got broken in their app and it doesn't allow saving any data. When I use Habits to export the backup, I remember choosing to remember the choice to use Syncthing. However, I cannot find a way to change it back to other method of exporting the backup, e.g., using the file manager. Is there a way to do this somehow?

**Comment by navinthomas** [comment]
I think you'll have to clear the defaults associated with the specific app, which is Syncthing in your case. This should be a standard procedure.

### Helpful links
- https://www.lifewire.com/set-and-clear-android-default-apps-3886190#toc-how-to-clear-default-apps
